sdk/billingbenefits/Azure.ResourceManager.BillingBenefits/tests/Scenario/ReservationOrderAliasTests.cs

```diff
@@ -0,0 +1,220 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Collections.Generic;
+using System.Threading.Tasks;
+using Azure.Core.TestFramework;
+using Azure.ResourceManager.Resources;
+using NUnit.Framework;
+using Azure.ResourceManager.BillingBenefits.Models;
+using Azure.Core;
+
+namespace Azure.ResourceManager.BillingBenefits.Tests
+{
+    public class ReservationOrderAliasTests : BillingBenefitsManagementTestBase
+    {
+        private ReservationOrderAliasModelResource ModelResource { get; set; }
+        private TenantResource tenantResource { get; set; }
```

Comment:
I think it's better to use field here.
```suggestion
        private ReservationOrderAliasModelResource _modelResource;
        private TenantResource _tenantResource;
```